oggm/cfg.py

```diff
@@ -253,6 +253,13 @@ def _log_param_change(self, key, value):
 _doc = 'The historical monthly climate timeseries stored in a netCDF file.'
 BASENAMES['climate_historical'] = ('climate_historical.nc', _doc)
 
+# so far, this is only ERA5_daily and does not work with the default OGGM
+# mass balance module
+_doc = 'The historical daily climate timeseries stored in a netCDF file. \
+    (only temperature is really changing on daily basis, precipitation is just\
+     assumed constant for every day)'
+BASENAMES['climate_historical_daily'] = ('climate_historical_daily.nc', _doc)
```

Comment:
Can you use parentheses to snitch multi-line strings instead of `\` ? (see examples above)